dcparser/dcparser.py

```diff
@@ -27,7 +27,7 @@ def load_denial_constraints(self, fpath):
         """
         tic = time.clock()
         if not self.ds.raw_data:
-            status = 'NO dataset specified'
+            status = 'No dataset specified'
```

Comment:
No need to address in this PR but we should maybe consider getting rid of returning `status` and `time` and instead just directly use `logging` messages.